web/common/DayRegulationInfo.js

```diff
@@ -143,15 +143,9 @@ export function DayRegulationInfo({
 
       <RegulationCheck
         key={4}
-        check={
-          checkBreakRespect &&
-          checkBreakRespect.status === RULE_RESPECT_STATUS.failure
-            ? {
-                status: RULE_RESPECT_STATUS.failure,
-                rule: ALERT_TYPES.maximumUninterruptedWorkTime
-              }
```

Comment:
Why was this removed?

Reply:
So for this one I think it'd be wise to have a talk with Raphael:
Before, rule 4 was considered automatically broken if rule 3 was broken.
rule 3 is about checking we had sufficient break time during the day (apply only if more than 6h of work)
rule 4 is about checking we don't have more than 6 consecutive hours without a break during the day.
To me it doesn't make sense to say that rule 3 broken => rule 4 broken. That's why I removed this part

Reply:
To be discussed with @rtaieb

Reply:
I agree with @tristan-gueguen , I don't see why rule 3 was a prerequesite for rule 4.